server/sonar-server/src/main/java/org/sonar/server/computation/measure/MeasureRepositoryImpl.java

```diff
@@ -103,6 +91,11 @@ public void update(Component component, Metric metric, Measure measure) {
     delegate.update(component, metric, measure);
   }
 
+  public Set<Measure> getRawMeasures(Component component, Metric metric) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)